### ISSUE TITLE
[Config Support]: Recordings showing black screen

### ISSUE THREAD

**Issue by BurningTheSky** [blakeblackshear/frigate#8888]
### Describe the problem you are having

I've had recordings from my Hikvision camera working without issue until today when I removed 2 detection models and now the recordings are just a black screen. There's audio but no video. The live view has sound and video. I can't see if I've accidentally deleted something else.

### Version

0.12.1-367d724

### Frigate config file

```yaml
go2rtc:
  streams:
    Front:
      - "ffmpeg:rtsp://user:password@192.168.x.x:554//Streaming/Channels/101/#video=copy#audio=aac"

cameras:

  Front: # <------ Name the camera
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/Front?video&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://127.0.0.1:8554/Front247?video
          input_args: preset-rtsp-restream
          roles:
            - detect
          hwaccel_args: preset-nvidia-h264
      output_args:
        record: preset-record-generic-audio-aac
    detect:
      enabled: True # <---- disable detection until you have a working camera feed
      width: 640 # <---- update for your camera's resolution
      height: 360 # <---- update for your camera's resolution
      fps: 7
    objects:
      track:
        - person
        - bicycle
        - car
        - motorcycle
        - cat
        - dog
    record:
      enabled: True
      retain:
          days: 3
          mode: all
      events:
        pre_capture: 5
        post_capture: 5
        retain:
          default: 42
          mode: active_objects
    motion:
      mask:    
        - 595,0,640,0,640,87,619,72,588,46,549,23,497,0
```

### Relevant log output

```shell
I can't see any errors, and I'm not sure what I should be looking for
```

### Operating system

UNRAID

### Install method

Docker Compose

### Coral version

PCIe

**Comment by NickM-27** [comment]
ffprobe from the system page will also be helpful

**Comment by BurningTheSky** [comment]
Sorry for the delay, a hard restart of the container solved the issue initially so I thought it was solved but it has reoccured. There's no error in the browser that I can see. I didn't notice it happened a couple of days ago so I don't think the logs go far enough back to see the trigger. The recordings are being created as I can hear audio on the recording, there's just no image. 

Stupidly I just restarted the docker before getting the ffprobe data but this is it now:
```
fprobe Output
Stream 0:
Return Code: 0

Video:

Codec: H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10
Resolution: 3200x1800
FPS: 20/1

Audio:

Codec: AAC (Advanced Audio Coding)

Stream 1:
Return Code: 0

Video:

Codec: H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10
Resolution: 640x360
FPS: 8/1

Copy
```

**Comment by BurningTheSky** [comment]
Also, the thumbnails for events seem to be being created correctly

**Comment by NickM-27** [comment]
I'd suggest updating go2rtc https://docs.frigate.video/configuration/advanced#custom-go2rtc-version

**Comment by BurningTheSky** [comment]
Thanks, I'll give that a try. I've updated from 1.8.1 to 1.8.4 and will keep an eye on it

**Comment by BurningTheSky** [comment]
happened again, ffprobe doesn't seem out of shape:

```
Ffprobe Output
Stream 0:
Return Code: 0

Video:

Codec: H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10
Resolution: 3200x1800
FPS: 20/1

Audio:

Codec: AAC (Advanced Audio Coding)

Stream 1:
Return Code: 0

Video:

Codec: H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10
Resolution: 640x360
FPS: 8/1
```

I've tried going back to the version that comes with frigate to see if that helps. It seemed to happen when the container was restarted by the host after a backup but I don't see any issue in the log.

```
{"log":"2023-12-31 11:38:02.968255100  frame=  467 fps= 19 q=-1.0 size=N/A time=00:00:27.47 bitrate=N/A speed=1.09x    \rframe=  630 fps= 25 q=-1.0 size=N/A time=00:00:27.31 bitrate=N/A speed=1.06x    \rframe=  464 fps= 19 q=-1.0 size=N/A time=00:00:27.34 bitrate=N/A speed=1.09x    \rframe=  457 fps= 19 q=-1.0 size=N/A time=00:00:27.01 bitrate=N/A speed= 1.1x    \rframe=  353 fps= 14 q=-1.0 size=N/A time=00:00:27.62 bitrate=N/A speed=1.11x    \rframe=  349 fps= 14 q=-1.0 size=N/A time=00:00:27.40 bitrate=N/A speed=1.11x    \rframe=  477 fps= 19 q=-1.0 size=N/A time=00:00:27.91 bitrate=N/A speed=1.09x    \rframe=  475 fps= 19 q=-1.0 size=N/A time=00:00:27.78 bitrate=N/A speed=1.09x    \rframe=  644 fps= 25 q=-1.0 size=N/A time=00:00:27.82 bitrate=N/A speed=1.06x    \rframe=  467 fps= 19 q=-1.0 size=N/A time=00:00:27.52 bitrate=N/A speed= 1.1x    \rframe=  360 fps= 14 q=-1.0 size=N/A time=00:00:28.13 bitrate=N/A speed=1.11x    \rframe=  358 fps= 14 q=-1.0 size=N/A time=00:00:27.98 bitrate=N/A speed=1.11x    \rframe=  487 fps= 19 q=-1.0 size=N/A time=00:00:28.49 bitrate=N/A speed=1.09x    \rframe=  485 fps= 19 q=-1.0 size=N/A time=00:00:28.36 bitrate=N/A speed=1.09x    \rframe=  657 fps= 25 q=-1.0 size=N/A time=00:00:28.39 bitrate=N/A speed=1.06x    \rframe=  478 fps= 19 q=-1.0 size=N/A time=00:00:28.03 bitrate=N/A speed=1.09x    \rframe=  368 fps= 14 q=-1.0 size=N/A time=00:00:28.64 bitrate=N/A speed= 1.1x    \rframe=  367 fps= 14 q=-1.0 size=N/A time=00:00:28.55 bitrate=N/A speed=1.11x    \rframe=  495 fps= 19 q=-1.0 size=N/A time=00:00:28.87 bitrate=N/A speed=1.09x    \rframe=  669 fps= 25 q=-1.0 size=N/A time=00:00:28.91 bitrate=N/A speed=1.06x    \rframe=  498 fps= 19 q=-1.0 size=N/A time=00:00:28.94 bitrate=N/A speed=1.08x    \rframe=  489 fps= 19 q=-1.0 size=N/A time=00:00:28.54 bitrate=N/A speed=1.09x    \rframe=  377 fps= 14 q=-1.0 size=N/A time=00:00:29.22 bitrate=N/A speed= 1.1x    \rframe=  376 fps= 14 q=-1.0 size=N/A time=00:00:29.13 bitrate=N/A speed= 1.1x    \rframe=  504 fps= 19 q=-1.0 size=N/A time=00:00:29.32 bitrate=N/A speed=1.08x    \rframe=  507 fps= 19 q=-1.0 size=N/A time=00:00:29.45 bitrate=N/A speed=1.08x    \rframe=  679 fps= 25 q=-1.0 size=N/A time=00:00:29.29 bitrate=N/A speed=1.06x    \rframe=  498 fps= 19 q=-1.0 size=N/A time=00:00:29.06 bitrate=N/A speed=1.09x    \rframe=  385 fps= 14 q=-1.0 size=N/A time=00:00:29.73 bitrate=N/A speed=1.09x    \rframe=  385 fps= 14 q=-1.0 size=N/A time=00:00:29.77 bitrate=N/A speed= 1.1x    \rframe=  516 fps= 19 q=-1.0 size=N/A time=00:00:29.90 bitrate=N/A speed=1.08x    \rframe=  689 fps= 24 q=-1.0 size=N/A time=00:00:29.67 bitrate=N/A speed=1.05x    \rframe=  519 fps= 19 q=-1.0 size=N/A time=00:00:30.03 bitrate=N/A speed=1.08x    \rframe=  509 fps= 19 q=-1.0 size=N/A time=00:00:29.63 bitrate=N/A speed=1.09x    \rframe=  396 fps= 14 q=-1.0 size=N/A time=00:00:30.50 bitrate=N/A speed= 1.1x    \rframe=  391 fps= 14 q=-1.0 size=N/A time=00:00:30.22 bitrate=N/A speed= 1.1x    \rframe=  527 fps= 19 q=-1.0 size=N/A time=00:00:30.47 bitrate=N/A speed=1.08x    \rframe=  523 fps= 19 q=-1.0 size=N/A time=00:00:30.22 bitrate=N/A speed=1.07x    \rframe=  519 fps= 19 q=-1.0 size=N/A time=00:00:30.08 bitrate=N/A speed=1.09x    \rframe=  709 fps= 25 q=-1.0 size=N/A time=00:00:30.51 bitrate=N/A speed=1.06x    \rframe=  404 fps= 14 q=-1.0 size=N/A time=00:00:31.07 bitrate=N/A speed= 1.1x    \rframe=  401 fps= 14 q=-1.0 size=N/A time=00:00:30.86 bitrate=N/A speed= 1.1x    \rframe=  539 fps= 19 q=-1.0 size=N/A time=00:00:31.05 bitrate=N/A speed=1.08x    \rframe=  721 fps= 25 q=-1.0 size=N/A time=00:00:30.89 bitrate=N/A speed=1.05x    \rframe=  537 fps= 19 q=-1.0 size=N/A time=00:00:30.98 bitrate=N/A speed=1.08x    \rframe=  530 fps= 19 q=-1.0 size=N/A time=00:00:30.66 bitrate=N/A speed=1.08x    \r[h264 @ 0x562a94823200] error while decoding MB 193 101, bytestream -13\n","stream":"stdout","time":"2023-12-31T11:38:02.984555064Z"}
{"log":"2023-12-31 11:38:02.968266065  [h264 @ 0x562a94823200] concealing 2256 DC, 2256 AC, 2256 MV errors in P frame\n","stream":"stdout","time":"2023-12-31T11:38:02.984618899Z"}
{"log":"2023-12-31 11:38:03.129204516  frame=  410 fps= 14 q=-1.0 size=N/A time=00:00:31.46 bitrate=N/A speed=1.09x    \r[h264 @ 0x562a94823200] Increasing reorder buffer to 1\n","stream":"stdout","time":"2023-12-31T11:38:03.132185623Z"}
{"log":"2023-12-31 11:38:03.815838883  frame=  547 fps= 19 q=-1.0 size=N/A time=00:00:31.43 bitrate=N/A speed=1.08x    \rframe=  549 fps= 19 q=-1.0 size=N/A time=00:00:31.56 bitrate=N/A speed=1.07x    \rframe=  409 fps= 14 q=-1.0 size=N/A time=00:00:31.37 bitrate=N/A speed= 1.1x    \rframe=  733 fps= 25 q=-1.0 size=N/A time=00:00:31.47 bitrate=N/A speed=1.05x    \rframe=  540 fps= 19 q=-1.0 size=N/A time=00:00:31.17 bitrate=N/A speed=1.08x    \rframe=  420 fps= 14 q=-1.0 size=N/A time=00:00:32.10 bitrate=N/A speed=1.09x    \rframe=  561 fps= 19 q=-1.0 size=N/A time=00:00:32.14 bitrate=N/A speed=1.08x    \rframe=  550 fps= 19 q=-1.0 size=N/A time=00:00:31.68 bitrate=N/A speed=1.08x    \rframe=  559 fps= 19 q=-1.0 size=N/A time=00:00:32.01 bitrate=N/A speed=1.08x    \rframe=  748 fps= 25 q=-1.0 size=N/A time=00:00:31.98 bitrate=N/A speed=1.05x    \rframe=  418 fps= 14 q=-1.0 size=N/A time=00:00:32.01 bitrate=N/A speed= 1.1x    \rInput #0, rtsp, from 'rtsp://user:password@192.168.x.54:554//Streaming/Channels/101/':\n","stream":"stdout","time":"2023-12-31T11:38:03.828594667Z"}
{"log":"2023-12-31 11:38:03.815846915    Metadata:\n","stream":"stdout","time":"2023-12-31T11:38:03.828633429Z"}
{"log":"2023-12-31 11:38:03.815852083      title           : Media Presentation\n","stream":"stdout","time":"2023-12-31T11:38:03.828709137Z"}
{"log":"2023-12-31 11:38:03.815856902    Duration: N/A, start: 0.000000, bitrate: N/A\n","stream":"stdout","time":"2023-12-31T11:38:03.828722337Z"}
{"log":"2023-12-31 11:38:03.815862070    Stream #0:0: Video: h264 (Main), yuvj420p(pc, progressive), 3200x1800 [SAR 1:1 DAR 16:9], 20 fps, 25 tbr, 90k tbn\n","stream":"stdout","time":"2023-12-31T11:38:03.828733302Z"}
{"log":"2023-12-31 11:38:03.815866819    Stream #0:1: Audio: aac (LC), 48000 Hz, stereo, fltp\n","stream":"stdout","time":"2023-12-31T11:38:03.828744198Z"}
{"log":"2023-12-31 11:38:03.815871988  Stream mapping:\n","stream":"stdout","time":"2023-12-31T11:38:03.828756909Z"}
{"log":"2023-12-31 11:38:03.815876737    Stream #0:0 -\u003e #0:0 (copy)\n","stream":"stdout","time":"2023-12-31T11:38:03.828767804Z"}
{"log":"2023-12-31 11:38:03.815881975    Stream #0:1 -\u003e #0:1 (aac (native) -\u003e aac (native))\n","stream":"stdout","time":"2023-12-31T11:38:03.828779607Z"}
{"log":"2023-12-31 11:38:03.815886654  Press [q] to stop, [?] for help\n","stream":"stdout","time":"2023-12-31T11:38:03.828790502Z"}
{"log":"2023-12-31 11:38:04.105010513  frame=  426 fps= 14 q=-1.0 size=N/A time=00:00:32.48 bitrate=N/A speed=1.09x    \r11:38:04.104 DBG [exec] run launch=5.389393231s\n","stream":"stdout","time":"2023-12-31T11:38:04.120395833Z"}
{"log":"2023-12-31 11:38:04.105668142  Output #0, rtsp, to 'rtsp://127.0.0.1:8554/59d9f18278623e17b5ffe7527c9c6681':\n","stream":"stdout","time":"2023-12-31T11:38:04.120433687Z"}
{"log":"2023-12-31 11:38:04.105750415    Metadata:\n","stream":"stdout","time":"2023-12-31T11:38:04.120466792Z"}
{"log":"2023-12-31 11:38:04.105860695      title           : Media Presentation\n","stream":"stdout","time":"2023-12-31T11:38:04.12048055Z"}
{"log":"2023-12-31 11:38:04.105969508      encoder         : Lavf59.27.100\n","stream":"stdout","time":"2023-12-31T11:38:04.120491935Z"}
{"log":"2023-12-31 11:38:04.106233509    Stream #0:0: Video: h264 (Main), yuvj420p(pc, progressive), 3200x1800 [SAR 1:1 DAR 16:9], q=2-31, 20 fps, 25 tbr, 90k tbn\n","stream":"stdout","time":"2023-12-31T11:38:04.120502271Z"}
{"log":"2023-12-31 11:38:04.106379688    Stream #0:1: Audio: aac (LC), 48000 Hz, stereo, fltp, 128 kb/s\n","stream":"stdout","time":"2023-12-31T11:38:04.120513725Z"}
{"log":"2023-12-31 11:38:04.106450647      Metadata:\n","stream":"stdout","time":"2023-12-31T11:38:04.120526436Z"}
{"log":"2023-12-31 11:38:04.106564279        encoder         : Lavc59.37.100 aac\n","stream":"stdout","time":"2023-12-31T11:38:04.120536354Z"}
{"log":"2023-12-31 11:38:07.853812617  192.168.x.45 - - [31/Dec/2023:11:38:07 +0000] \"GET /api/stats HTTP/1.1\" 200 4830 \"-\" \"HomeAssistant/2023.12.4 aiohttp/3.9.1 Python/3.11\" \"-\"\n","stream":"stdout","time":"2023-12-31T11:38:07.86213856Z"}
{"log":"2023-12-31 11:38:12.043631848  frame=    0 fps=0.0 q=-1.0 size=N/A time=00:00:00.00 bitrate=N/A speed=   0x    \rframe=  571 fps= 19 q=-1.0 size=N/A time=00:00:32.65 bitrate=N/A speed=1.07x    \rframe=    0 fps=0.0 q=-1.0 size=N/A time=00:00:04.75 bitrate=N/A speed=9.42x    \rframe=  427 fps= 14 q=-1.0 size=N/A time=00:00:32.52 bitrate=N/A speed= 1.1x    \rframe=  561 fps= 19 q=-1.0 size=N/A time=00:00:32.26 bitrate=N/A speed=1.08x    \rframe=  762 fps= 25 q=-1.0 size=N/A time=00:00:32.55 bitrate=N/A speed=1.05x    \rframe=  570 fps= 19 q=-1.0 size=N/A time=00:00:32.58 bitrate=N/A speed=1.08x    \rframe=  581 fps= 19 q=-1.0 size=N/A time=00:00:33.16 bitrate=N/A speed=1.07x    \rframe=  433 fps= 14 q=-1.0 size=N/A time=00:00:32.97 bitrate=N/A speed=1.09x    \rframe=  773 fps= 25 q=-1.0 size=N/A time=00:00:33.07 bitrate=N/A speed=1.05x    \rframe=  580 fps= 19 q=-1.0 size=N/A time=00:00:33.10 bitrate=N/A speed=1.07x    \rframe=  436 fps= 14 q=-1.0 size=N/A time=00:00:33.19 bitrate=N/A speed=1.09x    \rframe=   10 fps=9.7 q=-1.0 size=N/A time=00:00:05.59 bitrate=N/A speed=5.44x    \rframe=  573 fps= 19 q=-1.0 size=N/A time=00:00:32.83 bitrate=N/A speed=1.08x    \rframe=  592 fps= 19 q=-1.0 size=N/A time=00:00:33.67 bitrate=N/A speed=1.07x    \rframe=  591 fps= 19 q=-1.0 size=N/A time=00:00:33.61 bitrate=N/A speed=1.07x    \rframe=  788 fps= 25 q=-1.0 size=N/A time=00:00:33.64 bitrate=N/A speed=1.05x    \rframe=  442 fps= 14 q=-1.0 size=N/A time=00:00:33.54 bitrate=N/A speed=1.09x    \rframe=  446 fps= 14 q=-1.0 size=N/A time=00:00:33.89 bitrate=N/A speed=1.09x    \rframe=  583 fps= 19 q=-1.0 size=N/A time=00:00:33.28 bitrate=N/A speed=1.08x    \rframe=   22 fps= 14 q=-1.0 size=N/A time=00:00:06.14 bitrate=N/A speed=3.93x    \rframe=  602 fps= 19 q=-1.0 size=N/A time=00:00:34.19 bitrate=N/A speed=1.07x    \rframe=  797 fps= 25 q=-1.0 size=N/A time=00:00:33.96 bitrate=N/A speed=1.05x    \rframe=  448 fps= 14 q=-1.0 size=N/A time=00:00:34.06 bitrate=N/A speed=1.09x    \rframe=  601 fps= 19 q=-1.0 size=N/A time=00:00:34.18 bitrate=N/A speed=1.07x    \rframe=  593 fps= 19 q=-1.0 size=N/A time=00:00:33.86 bitrate=N/A speed=1.08x    \rframe=   31 fps= 14 q=-1.0 size=N/A time=00:00:06.65 bitrate=N/A speed=2.98x    \rframe=  451 fps= 14 q=-1.0 size=N/A time=00:00:34.21 bitrate=N/A speed=1.07x    \rframe=  455 fps= 14 q=-1.0 size=N/A time=00:00:34.44 bitrate=N/A speed=1.08x    \rframe=  611 fps= 19 q=-1.0 size=N/A time=00:00:34.63 bitrate=N/A speed=1.07x    \rframe=  613 fps= 19 q=-1.0 size=N/A time=00:00:34.70 bitrate=N/A speed=1.07x    \rframe=  604 fps= 19 q=-1.0 size=N/A time=00:00:34.37 bitrate=N/A speed=1.07x    \rframe=  813 fps= 25 q=-1.0 size=N/A time=00:00:34.60 bitrate=N/A speed=1.04x    \rframe=   43 fps= 16 q=-1.0 size=N/A time=00:00:07.23 bitrate=N/A speed=2.61x    \rframe=  465 fps= 14 q=-1.0 size=N/A time=00:00:35.11 bitrate=N/A speed=1.09x    \rframe=  623 fps= 19 q=-1.0 size=N/A time=00:00:35.27 bitrate=N/A speed=1.07x    \rframe=  622 fps= 19 q=-1.0 size=N/A time=00:00:35.21 bitrate=N/A speed=1.07x    \rframe=  465 fps= 14 q=-1.0 size=N/A time=00:00:35.14 bitrate=N/A speed=1.09x    \rframe=  614 fps= 19 q=-1.0 size=N/A time=00:00:34.88 bitrate=N/A speed=1.07x    \rframe=  828 fps= 25 q=-1.0 size=N/A time=00:00:35.24 bitrate=N/A speed=1.05x    \rframe=   50 fps= 15 q=-1.0 size=N/A time=00:00:07.59 bitrate=N/A speed= 2.3x    \rframe=  473 fps= 14 q=-1.0 size=N/A time=00:00:35.68 bitrate=N/A speed=1.09x    \rframe=  633 fps= 19 q=-1.0 size=N/A time=00:00:35.79 bitrate=N/A speed=1.07x    \rframe=  472 fps= 14 q=-1.0 size=N/A time=00:00:35.59 bitrate=N/A speed=1.08x    \rframe=  632 fps= 19 q=-1.0 size=N/A time=00:00:35.66 bitrate=N/A speed=1.07x    \rframe=  625 fps= 19 q=-1.0 size=N/A time=00:00:35.39 bitrate=N/A speed=1.07x    \rframe=  844 fps= 25 q=-1.0 size=N/A time=00:00:35.88 bitrate=N/A speed=1.05x    \rframe=  479 fps= 14 q=-1.0 size=N/A time=00:00:36.13 bitrate=N/A speed=1.08x    \rframe=   65 fps= 16 q=-1.0 size=N/A time=00:00:08.34 bitrate=N/A speed=2.09x    \rframe=  643 fps= 19 q=-1.0 size=N/A time=00:00:36.30 bitrate=N/A speed=1.07x    \rframe=  642 fps= 19 q=-1.0 size=N/A time=00:00:36.23 bitrate=N/A speed=1.07x    \rframe=  635 fps= 19 q=-1.0 size=N/A time=00:00:35.90 bitrate=N/A speed=1.07x    \rframe=  479 fps= 14 q=-1.0 size=N/A time=00:00:36.10 bitrate=N/A speed=1.08x    \rframe=  856 fps= 25 q=-1.0 size=N/A time=00:00:36.39 bitrate=N/A speed=1.05x    \rframe=  488 fps= 14 q=-1.0 size=N/A time=00:00:36.64 bitrate=N/A speed=1.08x    \rframe=   75 fps= 17 q=-1.0 size=N/A time=00:00:08.85 bitrate=N/A speed=1.96x    \rframe=  652 fps= 19 q=-1.0 size=N/A time=00:00:36.75 bitrate=N/A speed=1.06x    \rframe=  655 fps= 19 q=-1.0 size=N/A time=00:00:36.81 bitrate=N/A speed=1.07x    \rframe=  646 fps= 19 q=-1.0 size=N/A time=00:00:36.48 bitrate=N/A speed=1.07x    \rframe=  490 fps= 14 q=-1.0 size=N/A time=00:00:36.81 bitrate=N/A speed=1.08x    \rframe=  869 fps= 25 q=-1.0 size=N/A time=00:00:36.91 bitrate=N/A speed=1.05x    \rframe=  497 fps= 14 q=-1.0 size=N/A time=00:00:37.22 bitrate=N/A speed=1.08x    \rframe=  664 fps= 19 q=-1.0 size=N/A time=00:00:37.32 bitrate=N/A speed=1.06x    \rframe=   91 fps= 17 q=-1.0 size=N/A time=00:00:09.66 bitrate=N/A speed=1.84x    \rframe=  658 fps= 19 q=-1.0 size=N/A time=00:00:37.06 bitrate=N/A speed=1.07x    \rframe=  665 fps= 19 q=-1.0 size=N/A time=00:00:37.38 bitrate=N/A speed=1.06x    \rframe=  497 fps= 14 q=-1.0 size=N/A time=00:00:37.26 bitrate=N/A speed=1.08x    \rframe=  880 fps= 25 q=-1.0 size=N/A time=00:00:37.35 bitrate=N/A speed=1.04x    \rframe=  675 fps= 19 q=-1.0 size=N/A time=00:00:37.83 bitrate=N/A speed=1.06x    \rframe=  501 fps= 14 q=-1.0 size=N/A time=00:00:37.54 bitrate=N/A speed=1.07x    \rframe=  100 fps= 17 q=-1.0 size=N/A time=00:00:10.09 bitrate=N/A speed=1.73x    \rframe=  669 fps= 19 q=-1.0 size=N/A time=00:00:37.63 bitrate=N/A speed=1.07x    \rframe=  677 fps= 19 q=-1.0 size=N/A time=00:00:38.02 bitrate=N/A speed=1.07x    \rframe=  898 fps= 25 q=-1.0 size=N/A time=00:00:37.99 bitrate=N/A speed=1.04x    \rframe=  507 fps= 14 q=-1.0 size=N/A time=00:00:37.96 bitrate=N/A speed=1.08x    \rframe=  511 fps= 14 q=-1.0 size=N/A time=00:00:38.24 bitrate=N/A speed=1.07x    \rframe=  685 fps= 19 q=-1.0 size=N/A time=00:00:38.35 bitrate=N/A speed=1.06x    \rframe=  688 fps= 19 q=-1.0 size=N/A time=00:00:38.54 bitrate=N/A speed=1.06x    \rframe=  680 fps= 19 q=-1.0 size=N/A time=00:00:38.14 bitrate=N/A speed=1.07x    \rframe=  114 fps= 18 q=-1.0 size=N/A time=00:00:10.81 bitrate=N/A speed=1.67x    \rframe=  511 fps= 14 q=-1.0 size=N/A time=00:00:38.22 bitrate=N/A speed=1.07x    \rframe=  912 fps= 25 q=-1.0 size=N/A time=00:00:38.63 bitrate=N/A speed=1.04x    \rframe=  695 fps= 19 q=-1.0 size=N/A time=00:00:38.86 bitrate=N/A speed=1.06x    \rframe=  522 fps= 14 q=-1.0 size=N/A time=00:00:38.95 bitrate=N/A speed=1.08x    \rframe=  698 fps= 19 q=-1.0 size=N/A time=00:00:39.05 bitrate=N/A speed=1.06x    \rframe=  690 fps= 19 q=-1.0 size=N/A time=00:00:38.66 bitrate=N/A speed=1.06x    \rframe=  129 fps= 18 q=-1.0 size=N/A time=00:00:11.54 bitrate=N/A speed=1.64x    \rframe=  524 fps= 14 q=-1.0 size=N/A time=00:00:39.11 bitrate=N/A speed=1.08x    \rframe=  705 fps= 19 q=-1.0 size=N/A time=00:00:39.37 bitrate=N/A speed=1.06x    \rframe=  530 fps= 14 q=-1.0 size=N/A time=00:00:39.46 bitrate=N/A speed=1.08x    \rframe=  927 fps= 25 q=-1.0 size=N/A time=00:00:39.27 bitrate=N/A speed=1.04x    \rframe=  709 fps= 19 q=-1.0 size=N/A time=00:00:39.56 bitrate=N/A speed=1.06x    \rframe=  701 fps= 19 q=-1.0 size=N/A time=00:00:39.23 bitrate=N/A speed=1.07x    \rframe=  142 fps= 19 q=-1.0 size=N/A time=00:00:12.14 bitrate=N/A speed= 1.6x    \rframe=  533 fps= 14 q=-1.0 size=N/A time=00:00:39.69 bitrate=N/A speed=1.08x    \rframe=  538 fps= 14 q=-1.0 size=N/A time=00:00:39.97 bitrate=N/A speed=1.07x    \rframe=  716 fps= 19 q=-1.0 size=N/A time=00:00:39.95 bitrate=N/A speed=1.06x    \rframe=  942 fps= 25 q=-1.0 size=N/A time=00:00:39.85 bitrate=N/A speed=1.04x    \rframe=  719 fps= 19 q=-1.0 size=N/A time=00:00:40.07 bitrate=N/A speed=1.06x    \rframe=  711 fps= 19 q=-1.0 size=N/A time=00:00:39.74 bitrate=N/A speed=1.06x    \rframe=  727 fps= 19 q=-1.0 size=N/A time=00:00:40.46 bitrate=N/A speed=1.06x    \rframe=  147\n","stream":"stdout","time":"2023-12-31T11:38:12.049071378Z"}
{"log":"2023-12-31 11:38:17.968182846  192.168.x.45 - - [31/Dec/2023:11:38:17 +0000] \"GET /api/stats HTTP/1.1\" 200 4832 \"-\" \"HomeAssistant/2023.12.4 aiohttp/3.9.1 Python/3.11\" \"-\"\n","stream":"stdout","time":"2023-12-31T11:38:17.983026544Z"}
```

**Comment by BurningTheSky** [comment]
Just noticed it happened again and I still can't see an error that stands out

**Comment by jay-kub** [comment]
Try changing your output_args to:

`record: preset-record-generic-audio-copy`

That fixed the issue for me as well as another user -- [#7995](https://github.com/blakeblackshear/frigate/issues/7995)

**Comment by BurningTheSky** [comment]
Thanks, I'll give that a try. I'm getting a weird clicking on the auto at the moment too which is fun!

**Comment by jay-kub** [comment]
The preset-record-generic-audio-aac option is actually for scenarios where your source audio _isn't_ AAC and therefore needs to be transcoded to AAC, and preset-record-generic-audio-copy is for scenarios where your source audio is AAC and the audio just needs to be copied.

So it's an easy mistake to make, and with it trying to transcode from AAC to AAC, it makes sense that there would be issues with the result.

Source: https://docs.frigate.video/configuration/ffmpeg_presets/#output-args-presets

**Comment by BurningTheSky** [comment]
It's happening again, and it's only for this one camera. It's happened maybe 3 times in the last month? And now I have to keep checking the files to see when it happens!

I've tried trimming down the logs to when the blank screen started: [https://pastebin.com/cQavVWmq](https://pastebin.com/cQavVWmq)

Current config for the relevant camera is:
```
go2rtc:
  log:
    exec: trace
  streams:
    Front:
     - "ffmpeg:rtsp://user:password@x.x.x.x:554//Streaming/Channels/101/#video=copy#audio=aac"
cameras:
  # <------- Event record ------->

  Front: # <------ Name the camera
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/Front?video&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://127.0.0.1:8554/Front247?video
          input_args: preset-rtsp-restream
          roles:
            - detect
          hwaccel_args: preset-nvidia-h264
      output_args:
        record: preset-record-generic-audio-copy # was preset-record-generic-audio-aac
    detect:
      enabled: True # <---- disable detection until you have a working camera feed
      width: 640 # <---- update for your camera's resolution
      height: 360 # <---- update for your camera's resolution
      fps: 7
    objects:
      track:
        - person
        - bicycle
        - car
        - motorcycle
        - cat
        - dog
    record:
      enabled: True
      retain:
          days: 3
          mode: all
      events:
        pre_capture: 5
        post_capture: 5
        retain:
          default: 42
          mode: active_objects
    motion:
      mask:    
        - 595,0,640,0,640,87,619,72,588,46,549,23,497,0
```

Currently on 0.13.2-6476f8a running go2rtc 1.8.4.

FFPROBE OUTPUT while in error:
```
Stream 0:
Return Code: 0

Video:

Codec: H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10
Resolution: 3200x1800
FPS: 20/1

Audio:

Codec: AAC (Advanced Audio Coding)

Stream 1:
Return Code: 0

Video:

Codec: H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10
Resolution: 640x360
FPS: 8/1
```

**Comment by NickM-27** [comment]
maybe try with this instead

```
go2rtc:
  streams:
    Front:
      - "rtsp://user:password@192.168.x.x:554//Streaming/Channels/101/"
      - "ffmpeg:Front#audio=aac"
```

**Comment by BurningTheSky** [comment]
Thanks, I'll give that a try!

**Comment by BurningTheSky** [comment]
mse for live and playback in Firefox for events has stopped working since making that change. The recordings work in vlc and webrtc and jsmpeg will play

**Comment by sgrossiP** [comment]
if anyone is here again. for me it is usually solved by restarting the camera. does not to seem a frigate issue

**Comment by Lallassu** [comment]
I had the same issue with 4x Unifi G3 bullet cameras. In the same Frigate setup I also have 4 TP-Link Vigi cameras and they work fine. I've noticed that at full hours it can suddenly stop recording and the recording is just black. Live feed works fine.
So for example recording works fine `10:59` but `11:00` it just stops. Sometimes it seems to be stopping after an event has been recorded (stopped as in black recording). Also, the recorded event is sometimes black as well. Guess that it is related to the current recorded video(?).

**Comment by jay-kub** [comment]
@NickM-27 is there anything additional we can provide to help fix this issue? I've been running into it since about December and nothing I've done has resolved the issue long-term. I've considered making a new post but decided against it, as it feels like it would be an exact duplicate of this one.

Issue: recordings are missing the video stream (black screen with audio). Logs are clear and there's no camera issues afaik--restream displays video correctly. I am restreaming Frigate to Synology Surveillance Station (SSS), and the corresponding recordings in SSS have a clear video stream. Frigate does appear to have a valid video stream but it appears it is getting lost during the conversion to an event recording.

The workaround for now is a nightly cron job that restarts the frigate container, as the recordings typically don't start to fail until Frigate has been up for a couple days.

**Comment by NickM-27** [comment]
> Frigate does appear to have a valid video stream but it appears it is getting lost during the conversion to an event recording.

frigate is only receiving audio from go2rtc, because when go2rtc reconnects to the camera the camera only sends audio first and the video track is not able to be added. From frigate's perspective there is nothing to be done here, the only option frigate would have is to restart the stream but that is what starts the problem in the first place.

You can just not restream the recording stream for frigate and instead have frigate connect to the camera directly.

**Comment by jay-kub** [comment]
Just to make sure I understand what you're saying--you're thinking it's a connectivity issue with the camera that then results in a missing video stream as the connection is being reset mid-recording?

**Comment by NickM-27** [comment]
all reports I have seen of this issue have been that after the recording stream is restarted due to a connection or other issue, the issue occurs.
